index.html

```diff
@@ -53,5 +53,144 @@ <h2 class=" role"> I am a Software developer</h2>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2" type="button">See Project</button>
+            </div>
+        
+        
+            <div class ="otherproject">
+            <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+            <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul id ="project2language">
+                <li class ="language2item">html</li>
+                <li class ="language2item" id ="bootstrap">bootstrap</li>
+                <li class ="language2item">Ruby</li>
+            </ul>
+            <button class="seeproject2" type="button">See Project</button>
+        </div>
+    
+        
+</section>
+    <section class="aboutsection" >
+        <div class ="about">
+
+            <h2 class="aboutheader"> About me</h2>
+            <p id ="aboutmetext">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button id="getresumebutton" type="button">Get My Resume</button>
+            
+        </div>
+        <hr class="divider">
+        <div class ="language">
+            <ul class ="skilllist">
+                <li class="lang" id="first">Language</li>
+                    <ul class="listitem">
+                        <li class="list">Javascript</li>
+                        <li class="list">Ruby</li>
+                        <li class="list">Html</li>
+                        <li class="list">Css</li>
+                    </ul>
+                    
+                </li>
+                <li class="lang" id="second">Frameworks</li>
+                    <ul class="listitem">
+                        <li class="list">Bootstrap</li>
+                        <li class="list">Ruby on Rails</li>
+                        <li class="list">RSPec</li>
+                        <li class="list">CapyBara</li>
+                        <li class="list">Selenium</li>
+                    </ul>
+                    
+                </li>
+                <li class="lang" id="third">Skills</li>
+                    <ul class="listitem">
+                        <li class="list">Codekit</li>
+                        <li class="list">Github</li>
+                        <li class="list">Codepen</li>
+                        <li class="list">Gitlab</li>
+                        <li class="list">Terminal</li>
+                    </ul>
+                    
+                </li>
+            </ul>
+
+        </div>
+        
```

Comment:
- [ ] Please make sure that the font family matches the design 
![image](https://user-images.githubusercontent.com/74744787/212491296-5cb7fbcd-cd87-420c-8b7e-3865ef183be0.png)
Figma design 
![image](https://user-images.githubusercontent.com/74744787/212491317-54c3c8be-ee95-410c-bfd9-8f3808ded270.png)